test/cookie.test.js

```diff
@@ -23,21 +21,17 @@ test('cookies get set correctly', (t) => {
     if (err) tap.error(err)
```

Comment:
`fastify.inject` does not require that the server be running and listening on a real port. So all of these `fastify.listen(0, (err) => {})` blocks can be unwrapped.